src/components/policy/policy_external/src/policy_manager_impl.cc

```diff
@@ -588,13 +588,15 @@ void PolicyManagerImpl::CheckPermissions(const PTString& app_id,
     policy_table::FunctionalGroupings functional_groupings;
     cache_->GetFunctionalGroupings(functional_groupings);
 
-#ifdef SDL_REMOTE_CONTROL
-    ApplicationOnDevice who = {device_id, app_id};
-    const policy_table::Strings app_groups = access_remote_->GetGroups(who);
-#else   // SDL_REMOTE_CONTROL
-    const policy_table::Strings app_groups =
+    policy_table::Strings app_groups =
         GetGroupsNames(app_group_permissions);
-#endif  // SDL_REMOTE_CONTROL
+
+//#ifdef SDL_REMOTE_CONTROL
+//    ApplicationOnDevice who = {device_id, app_id};
+//    const auto rc_groups = access_remote_->GetGroups(who);
```

Comment:
I think this code should just be removed. Any reason for keeping it commented out?